src/plugins/console/server/lib/spec_definitions/js/mappings.ts

```diff
@@ -84,7 +85,20 @@ export const mappings = (specService: SpecDefinitionsService) => {
           // index_prefixes: { min_chars, max_chars },
 
           index_options: {
-            __one_of: ['docs', 'freqs', 'positions'],
+            __one_of: [
```

Comment:
I think the autocomplete could work with the object just added among the string options, wdyt? 
```js
            __one_of: [
           'positions', 
           'docs', 
            'freqs', 
            'offsets',
            { 
              type: 
             ...
             }
           ]
```

Reply:
Thanks for this point. Yes, that's better, so I flattened the unnecessary nests of `__one_of`.